### PR TITLE
[INLONG-5967][Manager] Add save and delete interfaces of cluster management in OpenAPI

### LINKED ISSUE #5967: [Feature][Manager] Add save and delete interfaces of cluster management in OpenAPI
### Description

_No response_

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by vernedeng**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #5967

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

**Review by luchunliang — APPROVED** [review]
LGTM

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterService.java [hunk 1]
```diff
@@ -90,6 +91,16 @@ public interface InlongClusterService {
...
 
+    /**
+     * Save raw cluster info.
+     *
+     * This method should be repeatable.
+     *
+     * @param request raw inlong cluster info
+     * @return cluster id after saving
+     */
+    Integer saveRaw(InlongClusterEntity request);
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterService.java:102]: Please re-use the above save method.

Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterService.java:102]: Please re-use the above save method.

File: inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/openapi/OpenClusterController.java [hunk 2]
```diff
@@ -57,4 +59,17 @@ public Response<PageResult<ClusterInfo>> list(@RequestBody ClusterPageRequest re
         return Response.success(clusterService.list(request));
     }
 
+    @PostMapping(value = "/cluster/insert")
+    @ApiOperation(value = "insert raw cluster info")
+    public Response<Integer> insert(@RequestBody InlongClusterEntity request) {
```
Comment (healchow) [inline inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/openapi/OpenClusterController.java:64]: The controller layer should not use the entity from the dao layer directly.

**Review by healchow — CHANGES_REQUESTED** [review]

**Comment by vernedeng** [comment]
The registration process of sort-standalone task will be interageted into manager sdk, instead of openapi

**Comment by vernedeng** [comment]
the cluster managerment will be done by the users, instead of integrating it in inlong manager

**Pull request closed without merging**